tun/src/unix/linux/mod.rs

```diff
@@ -110,4 +132,17 @@ mod test {
 
         assert_eq!(interf.mtu().unwrap(), 500);
     }
+
+    #[test]
+    fn netmask() {
+        let interf = TunInterface::new().unwrap();
+
+        let netmask = Ipv4Addr::new(255, 0, 0, 0);
+        let addr = Ipv4Addr::new(192, 168, 1, 1);
+
+        interf.set_ipv4_addr(addr);
+        interf.set_netmask(netmask).unwrap();
+
+        assert_eq!(interf.netmask().unwrap(), netmask);
+    }
```

Comment:
```suggestion
    #[test]
    #[throws]
    fn netmask() {
        let interf = TunInterface::new()?;

        let netmask = Ipv4Addr::new(255, 0, 0, 0);
        let addr = Ipv4Addr::new(192, 168, 1, 1);

        interf.set_ipv4_addr(addr)?;
        interf.set_netmask(netmask)?;

        assert_eq!(interf.netmask()?, netmask);
    }
```
Could use `fehler::throws` to make it prettier, but looks great otherwise!